params/config.go

```diff
@@ -505,20 +505,21 @@ type ScrollConfig struct {
 	L1Config *L1Config `json:"l1Config,omitempty"`
 }
 
-// L1Config contains the l1 parameters needed to collect l1 messages in the sequencer
+// L1Config contains the l1 parameters needed to sync l1 contract events (e.g., l1 messages, commit/revert/finalize batches) in the sequencer
 type L1Config struct {
 	L1ChainId             uint64         `json:"l1ChainId,string,omitempty"`
 	L1MessageQueueAddress common.Address `json:"l1MessageQueueAddress,omitempty"`
 	NumL1MessagesPerBlock uint64         `json:"numL1MessagesPerBlock,string,omitempty"`
+	ScrollChainAddress    common.Address `json:"scrollChainAddress,omitempty"`
```

Comment:
Also update `DefaultScrollSepoliaGenesisBlock`

Reply:
changed in [7dcdbf6](https://github.com/scroll-tech/go-ethereum/pull/515/commits/7dcdbf6ec1b0e81490b2a1a28ddcc8b58f07bc53)